rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/TestCacheSize.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2018. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+package com.qubole.rubix.bookkeeper;
+
+import com.qubole.rubix.bookkeeper.utils.DiskUtils;
```

Comment:
We already have a test method `verifyCacheSizeMetricIsReported` in TestBookKeeper class. We should change the method.
Also if you want to write a different class, we don't refer to rubix core FSDataInputStream unless we definitely need to use that. We can use bookkeeper readdata method to download the data

Reply:
moving the unit test to TestBookKeeper